### DISCUSSION TITLE
AlDente - Stop charging when sleeping

### DISCUSSION THREAD

**Discussion by voriax** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1402] [General]
Hi!

I have one question about "Stop charging when sleeping" function.

What happens when this function is enabled & laptop goes down by accident to 0% ?

**Comment by MatthiasKerbl** [comment]
> Hi!
> 
> I have one question about "Stop charging when sleeping" function.
> 
> What happens when this function is enabled & laptop goes down by accident to 0% ?

Hi @voriax,

If your MacBook is plugged in and in sleep mode and charging is paused due to Stop Charging when Sleeping, you leave it like that for months so the battery auto discharges to 0%. The MacBook will turn off and due to AlDente not being active anymore when it is turned off, it will start charging again.

Best,
Matthias

**Reply by voriax** [reply]
Hello

Thanks for details. 

I was referring to situation that docking station connected via USBC instead of deliver will start to take out power from laptop, when main power for docking station is not delivered, I had it once :-(

This is why I got worried what would happen if AlDente stopped charging during sleep and then laptop went to 0 battery
